src/app/modules/group/http-services/get-requests.service.ts

```diff
@@ -27,13 +27,16 @@ export class GetRequestsService {
 
   getPendingRequests(
     groupId: string,
-    sort: string[] = []
+    sort: string[] = [],
+    includeSubgroup : boolean = false
```

Comment:
I would put the "sort" arg last as this is something not entirely related with the service and that may be common to many services.

Reply:
fixed